packages/ui/src/components/editor/input/Array/index.tsx

```diff
@@ -36,17 +38,44 @@ export interface ArrayInputProps {
   values: string[]
   onChange: (values: string[]) => void
   inputLabel?: string
+  dropTypes?: string[]
 }
 
-// TODO: file and drag and drop functionality
+const DiscardableInput = ({ value, index, inputLabel, onChange, onRemove, dropTypes }) => {
+  const [{ isDroppable }, dropRef] = useDrop(() => ({
+    accept: dropTypes ?? [...SupportedFileTypes],
+    drop: (item: { url: string }) => {
+      onChange(item.url, index)
+    },
+    collect: (monitor) => ({
+      isDroppable: monitor.canDrop() && monitor.isOver()
+    })
+  }))
+
+  return (
+    <div className="flex flex-col px-3">
+      {inputLabel && <Label className="mb-1 text-[#A0A1A2]">{inputLabel + ' ' + (index + 1)}</Label>}
+      <div ref={dropRef} className={`mb-2 flex items-center ${isDroppable ? 'outline outline-2 outline-white' : ''}`}>
```

Comment:
```suggestion
      <div ref={dropRef} className={twMerge('mb-2 flex items-center', isDroppable && 'outline outline-2 outline-white')}>
```

we can utilise twMerge in this cases - it simplifies using normal string